acj/static/modules/gradebook/gradebook-partial.html

```diff
@@ -47,7 +47,7 @@ <h2 class="col-sm-6">
                     <th class="text-center"><a href="" ng-click="reverse = predicate == 'num_answers' && !reverse; predicate='num_answers'">
                         Answered</a>
                     </th>
-                    <th class="text-center"><a href="" ng-click="reverse = !reverse; sortScore()">
+                    <th class="text-center" ng-show="includeScores"><a href="" ng-click="reverse = !reverse; sortScore()">
```

Comment:
maybe it is better to use ng-if instead of ng-show? otherwise, sortScore() is always evaluated.